SubmersedVR/Tweaks/ParticleFX.cs

```diff
@@ -0,0 +1,102 @@
+﻿using HarmonyLib;
+using UnityEngine;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.Collections;
+using FMODUnity;
+
+namespace SubmersedVR
+{
+    class ParticleFX
```

Comment:
Hm, could you add a comment what you tried to do here?
I guess you  tried to remove the obvious bill boarding that can happen in unity when you roll the head?

Reply:
I found some code from the VREnhancments guys https://github.com/IWhoI/SubnauticaVREnhancements/blob/master/VREnhancements/ParticleFX.cs and I was wondering if it added any benefit. I tried it and it didnt seem to help anything but left the code there in case you had any insight into what they were trying to do.